<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DasdcopyClass</class>
 <widget class="QDialog" name="DasdcopyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dasdcopy</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLineEdit" name="infile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>361</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="browseFileButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ckd File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shadowFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>361</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="browseSfButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shadow file</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>521</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLineEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>401</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="path">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>401</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="browseOutButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QComboBox" name="compressionComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zlib</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bz2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Compression</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lfsCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Build One Large File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="replaceCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace Output File If Exists</string>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>521</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
